modules/Microsoft.Web/sites/.test/webAppCommon/deploy.test.bicep

```
@@ -153,5 +153,6 @@ module testDeployment '../../deploy.bicep' = {
     userAssignedIdentities: {
       '${nestedDependencies.outputs.managedIdentityResourceId}': {}
     }
+    basicPublishingCredentialsPolicyName: 'ftp'
```

This is what I meant with array of objects
```suggestion
    basicPublishingCredentialsPolicies: [
      {
        name: 'ftp'
      }
      {
        name: 'scm'
      }
    ]
```

---

Updated as per your suggestion